### ISSUE TITLE
Unable to execute /bin/train_all.sh for Supervised embeddings

### ISSUE THREAD

**Issue by ahtsham58** [chaitjo/personalized-dialog#9]
Hi @chaitjo @MiFei,

I am in the process of reproducing the system. I am done with all the installations and now trying to execute ` supervised-embedding/bin/train_all.sh`.  I executed `build_experiments.py` and now trying to train the model but getting this error. I cannot see any .tsv files in my data. How to create these files ..?

![image](https://user-images.githubusercontent.com/23135687/74440154-c1520080-4e6d-11ea-8be1-94635039fb69.png)

I suspect there needs to run  "build_vocabulary.py" file before training. I also tried to run this script but the console stuck there without execution till 30 mints.
Please let me know what I am missing here or is there any issue with the code?

looking forward to your positive response.

**Comment by chaitjo** [comment]
Hi @ahtsham58, thanks for the interest. Unfortunately, its been so long since I worked on this project that my memory is a bit hazy. Are you using the correct version of TensorFlow--0.12.1? (For each of the models, refer to README inside their directory for exact requirements, usage, etc.)

Here is the repository from which I adapted the supervised embeddings approach: https://github.com/sld/supervised-embedding-model

Refer also to other closed issues such as https://github.com/chaitjo/personalized-dialog/issues/3

Hope it helps!
